lib/scrape.js

```diff
@@ -48,6 +49,23 @@ try {
   console.log('No existing data...')
 }
 
+const FEED_LINKS = [
+  'http://systers-opensource.blogspot.com/feeds/posts/default',
+  'https://www.haiku-os.org/blog/index.xml',
+  'https://blog.wikimedia.org/feed/',
+  'http://www.liquidgalaxylab.com/feeds/posts/default',
+  'https://blog.zulip.org/rss/',
+  'https://blog.musicbrainz.org/feed/',
+  'https://insights.ubuntu.com/feed/',
+  'http://www.xwiki.org/xwiki/bin/view/Main/WebRss?xpage=rdf',
+  'https://www.drupal.org/planet/rss.xml',
+  'http://planet.osgeo.org/atom.xml',
+  // Not formateed XML 'https://blog.coala.io/atom.xml',
+  'http://mifos.org/feed/',
+  'https://blog.fossasia.org/feed/',
+  'http://openmrs.org/feed/',
+]
```

Comment:
Ideally these would be fetched automatically, right? Using the `blog_url` key from the API and scraping for something like

```html
<link rel="alternate" type="application/atom+xml" href="/atom.xml" title="Feed">
```

Reply:
Right, I did fetch them, but there is a problem with coala's feed, can you check `https://blog.coala.io/atom.xml` on your machine?

Reply:
Yep, seems to not be working. But in that case just skip over it and only use working XML feeds. That'll be encouragement to get the coala feed fixed 😉

Reply:
I've created https://github.com/coala/devops/issues/92 to fix this, but that will be much later after this is merged.

Reply:
Seems it's fixed now😉